app/policies/grant_submission/submission_policy.rb

```diff
@@ -22,9 +23,9 @@ def initialize(context, scope)
 
     def resolve
       if user.in?(@grant.administrators) || user.system_admin?
-        GrantSubmission::Submission.includes(:submitter, :reviews).where(grant_id: @grant.id)
+        GrantSubmission::Submission.includes(:reviews, :applicants).where(grant_id: @grant.id)
```

Comment:
Should this include still include `:submitter`?

Reply:
I don't think so. As is now the submitter gets copied as an applicant on creation. If they remove themselves as an applicant, then they are done and no longer have access, but will have still be the submitter.